### ISSUE TITLE
Identify who last modified a record in A&R

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/bcparks-ar-admin#394]
#### Description:

As an admin, I want to be able to see who last modified a record in A&R so that I can better track and understand the change history of a record. 

Ability to track and see who last modified a record in A&R

#### Acceptance Criteria:

Given
When
Then 

Given
When
Then

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Wireframe(s):**

**Workflows:**

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
October 11, 2024 prioritization session: A&R team identified this as a new ticket/request for implementation.
 
Mural board: https://app.mural.co/t/bcparks2575/m/bcparks2575/1728427370222/4c9714df914c7ec8bdcf20a9ee1999cc9e664a27?wid=0-1728427872927

**Comment by nicolebottoms** [comment]
this doesn't even need to be included in the export, even if it was just available under the variance note, or at the bottom of the data entry page.

**Comment by nicolebottoms** [comment]
an example of where this would be useful:
this is entered in the variance note for boating; 

![Image](https://github.com/user-attachments/assets/69cc37a0-ae83-4829-8a0a-f409790f9c9d)
i dont know who has been entering this data, sometimes i think the PO does and sometimes i think the AS does; but with this information i would follow up with them to clarify when the multiplier is used, and that it isn't used for boat attendance numbers, but for boat attendance people
